pkg/controller/federatedtypeconfig/controller.go

```diff
@@ -173,22 +301,22 @@ func (c *FederatedTypeConfigController) startController(tc *corev1a1.FederatedTy
 		close(stopChan)
 		return fmt.Errorf("Error starting sync controller for %q: %v", kind, err)
 	}
-	log.Printf("Started sync controller for %s \n", kind)
+	glog.Infof("Started sync controller for %q \n", kind)
```

Comment:
(no action required) we should remove these benign newlines.

Reply:
Done.